test/core/test_lwt_list.ml

```diff
@@ -575,9 +580,132 @@ let suite = suite "lwt_list" [
   end;
+
+  test "iter_p big list" begin fun () ->
+    test_big_list Lwt_list.iter_p
+  end;
+
+  test "iter_s big list" begin fun () ->
+    test_big_list Lwt_list.iter_s
+  end;
+
+  test "iteri_p big list" begin fun () ->
+    let iter f = Lwt_list.iteri_p (fun _ x -> f x) in
+    test_big_list iter
+  end;
+
+  test "iteri_s big list" begin fun () ->
+    let iter f = Lwt_list.iteri_s (fun _ x -> f x) in
+    test_serialization iter
+  end;
+
+  test "map_p big list" begin fun () ->
+    test_big_list Lwt_list.map_p
+  end;
+
+  test "map_s big list" begin fun () ->
+    test_serialization Lwt_list.map_s
+  end;
+
+  test "mapi_p big list" begin fun () ->
+    let m f = Lwt_list.mapi_p (fun _ x -> f x) in
+    test_big_list m
+  end;
+
+  test "mapi_s big list" begin fun () ->
+    let m f = Lwt_list.mapi_s (fun _ x -> f x) in
+    test_big_list m
+  end;
+
+  test "rev_map_p big list" begin fun () ->
+    test_big_list Lwt_list.rev_map_p
+  end;
+
+  test "rev_map_s big list" begin fun () ->
+    test_big_list Lwt_list.rev_map_s
+  end;
+
+  test "fold_left_s big list" begin fun () ->
+    let m f = Lwt_list.fold_left_s (fun _ x -> f x >>= fun _ -> Lwt.return ()) () in
+    test_big_list m
+  end;
+
+  test "fold_right_s big list" begin fun () ->
+    let m f l = Lwt_list.fold_right_s (fun x _ -> f x >>= fun _ -> Lwt.return ()) l () in
+    test_big_list m
+  end;
+
+  test "filter_map_p big list" begin fun () ->
+    let m f = Lwt_list.filter_map_p (fun x -> f x >>= fun u -> Lwt.return (Some u)) in
+    test_big_list m
+  end;
+
+  test "filter_map_s serlialism" begin fun () ->
+    let m f = Lwt_list.filter_map_s (fun x -> f x >>= fun u -> Lwt.return (Some u)) in
+    test_big_list m
+  end;
+
+  test "for_all_p big list" begin fun () ->
+    let m f = Lwt_list.for_all_p (fun x -> f x >>= fun _ -> Lwt.return_true) in
+    test_big_list m
+  end;
+
+  test "for_all_s big list" begin fun () ->
+    let m f = Lwt_list.for_all_s (fun x -> f x >>= fun _ -> Lwt.return_true) in
+    test_big_list m
+  end;
+
+  test "exists_p big list" begin fun () ->
+    let m f = Lwt_list.exists_p (fun x -> f x >>= fun _ -> Lwt.return_false) in
+    test_big_list m
+  end;
+
+  test "exists_s big list" begin fun () ->
+    let m f = Lwt_list.exists_s (fun x -> f x >>= fun _ -> Lwt.return_false) in
+    test_big_list m
+  end;
+
+  test "find_s big list" begin fun () ->
+    let m f = Lwt_list.find_s (fun x -> f x >>= fun _ -> Lwt.return_false) in
+    let handler e =
+      if e = Not_found then Lwt.return_true
+      else Lwt.return_false
+    in
+    Lwt.catch (fun () -> test_big_list m) handler
+  end;
+
+  test "filter_p big list" begin fun () ->
+    let m f = Lwt_list.filter_p (fun x -> f x >>= fun _ -> Lwt.return_true) in
+    test_big_list m
+  end;
+
+  test "filter_s big list" begin fun () ->
+    let m f = Lwt_list.filter_s (fun x -> f x >>= fun _ -> Lwt.return_true) in
+    test_big_list m
+  end;
+
+  test "filter_map_p big list" begin fun () ->
```

Comment:
I think this test and the next one are duplicates of two tests above.

Reply:
Good catch, thanks.